### PR TITLE
Remove supervisor-stdout as a dependency

### PR DESCRIPTION
**Pull request by kleesc**
### Description of Changes

* Removes supervisor-stdout as dependency. It doesn't look like the repo is being maintained. The source has been updated to work on Python 3 but hasn't been published to pypi. 

#### Issue: <link to story or task>

**TESTING** ->

**BREAKING CHANGE** ->

---

## Reviewer Checklist

- [ ] It works!
- [ ] Comments provide sufficient explanations for the next contributor
- [ ] Tests cover changes and corner cases
- [ ] Follows Quay syntax patterns and format

### CODE REVIEW

File: conf/supervisord.conf.jnj [hunk 1]
```diff
@@ -257,4 +349,6 @@ command=python -m workers.repomirrorworker.repomirrorworker
 autostart = {{ config['repomirrorworker']['autostart'] }}
 stdout_events_enabled = true
 stderr_events_enabled = true
-# EOF NO NEWLINE
```
Comment (thomasmckay) [inline conf/supervisord.conf.jnj]: Please keep this; no newline at eof in this file.

File: config_app/conf/supervisord.conf [hunk 2]
```diff
@@ -13,14 +11,6 @@ serverurl=unix:///%(ENV_QUAYDIR)s/config_app/conf/supervisord.sock
...
 supervisor.rpcinterface_factory = supervisor.rpcinterface:make_main_rpcinterface
 
-[eventlistener:stdout]
-environment=
-  PYTHONPATH=%(ENV_QUAYDIR)s
-command = supervisor_stdout
-buffer_size = 1024
-events = PROCESS_LOG
-result_handler = supervisor_stdout:event_handler
-
```
Comment (thomasmckay) [inline config_app/conf/supervisord.conf]: The config app should have all process go to stdout as well.

Comment (kleesc) [reply config_app/conf/supervisord.conf]: They should already be:
```
stdout_logfile=/dev/stdout
stdout_logfile_maxbytes=0
stderr_logfile=/dev/stdout
stderr_logfile_maxbytes=0
```

File: conf/supervisord.conf.jnj (hunk 1 above)
Comment (kleesc) [reply conf/supervisord.conf.jnj]: Is that a comment or something used by supervisor?

**Comment by thomasmckay** [comment]
@kleesc Can you confirm that the output is still labeled with the process and timestamped? I think that is what supervisor-stdout is doing, in which case I am against getting rid of it.

**Pull request closed without merging**